gdcdictionary/schemas/subject.yaml

```diff
@@ -0,0 +1,559 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "subject"
+title: Subject
+type: object
+namespace: https://chicagoland.pandemicresponsecommons.org
+category: administrative
+program: '*'
+project: '*'
+description: >
+  The collection of all data related to a specific subject in the
+  context of a specific experiment.
+additionalProperties: false
+submittable: true
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+  - state
+
+links:
+  - exclusive: false
+    required: true
+    subgroup:
+    - name: studies
+      backref: subjects
+      label: member_of
+      target_type: study
+      multiplicity: many_to_many
+      required: false
+    - name: projects
+      backref: subjects
+      label: data_from
+      target_type: project
+      multiplicity: many_to_one
+      required: false
+
+required:
+  - submitter_id
+  - type
+
+uniqueKeys:
+  - [id]
+  - [project_id, submitter_id]
+
+# Case properties
+properties:
+  $ref: "_definitions.yaml#/ubiquitous_properties"
+
+  species:
+    description: "Taxonomic species of the subject."
+    enum:
+     - Drosophila melanogaster
+     - Homo sapiens
+     - Mus musculus
+     - Mustela putorius furo
+     - Rattus rattus
+     - Sus scrofa
+    termDef:
+      - term: species
+        source: NCI Thesaurus
+        term_id: C45293
+        term_version: 20.03e (Release date:2020-03-31)
+
+  strain:
+    description: >
+      A lower-level taxonomic rank used in microbiology or virology, plants and rodents, usually at the intraspecific level (within a species)
+    type: string
+    termDef:
+      - term: strain
+        source: NCI Thesaurus
+        term_id: C14419
+        term_version: 20.03e (Release date:2020-03-31)
+
+  taxon_id:
+    description: >
+      NCBI identifier for classification of organisms.
+    type: integer
+    termDef:
+      - term: taxon_id
+        source: NCI Thesaurus
+        term_id: C54539
+        term_version: 20.03e (Release date:2020-03-31)
+
+  genus:
```

Comment:
Didn't we decide that we would be removing genus from the subject node?

Reply:
Yes, it was decided to delete it from the Subject node